app/assets/stylesheets/modules/links.scss

```diff
@@ -0,0 +1,88 @@
+@import "variables";
+
+a:link, a:visited, a:active {
+  text-decoration: none;
+  color: $white;
+  text-decoration:none;
```

Comment:
Property `text-decoration` already defined on line 4<br>Colon after property should be followed by one space